### PR TITLE
config.md: some minor format fix

### PR DESCRIPTION
**Pull request by coolljt0725**
Signed-off-by: Lei Jitang <leijitang@huawei.com>

Remove unnecessary blank line.

ping @philips

### CODE REVIEW

**Comment by wking** [comment]
I think removing the empty line in the ChainID section makes sense,
but I'd rather keep the empty line in the DiffID section.  That
separates the normative DiffID lines(above the current blank) from the
informative note (below the current blank).

**Comment by coolljt0725** [comment]
@wking  
>but I'd rather keep the empty line in the DiffID section.  That
separates the normative DiffID lines(above the current blank) from the
informative note (below the current blank).

I'd like keep them together since they are both the description of DiffID and to keep consistent with other description like this.

File: config.md [hunk 1]
```diff
@@ -25,7 +25,6 @@ Changing it means creating a new derived image, instead of changing the existing
 
 A layer DiffID is a SHA256 digest over the layer's uncompressed tar archive and serialized in the descriptor digest format, e.g., `sha256:a9561eb1b190625c9adb5a9513e72c4dedafc1cb2d4c5236c9a6957ec7dfd5a9`.
 Layers must be packed and unpacked reproducibly to avoid changing the layer DiffID, for example by using tar-split to save the tar headers.
-
```
Comment (vbatts) [inline config.md]: keep this one. If anything make the NOTE below be italics.

**Comment by coolljt0725** [comment]
@vbatts  @wking  @philips  updated

**Comment by wking** [comment]
d25e4a765 looks good to me.

**Comment by philips** [comment]
LGTM

**Comment by jonboulle** [comment]
LGTM, whynot

**Pull request merged**